charts/fdb-operator/Chart.yaml

```diff
@@ -14,8 +14,21 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.0
+version: 0.24.5
```

Comment:
This is the chart version and not the `appVersion` so it doesn't have to match the operator version.

Reply:
Agreed, and it'll never match, since inevitably changes will need to be made to the chart without changing the operator version. I felt that continuity would be a good place to start though. Happy to change this as required!